### ISSUE TITLE
FacilityID Attribute Rule Deprecation

### ISSUE THREAD

**Issue by rsparker-utah** [uic-utah/uic-attribute-rules#124]
Upon the completion of issue [#259](https://github.com/uic-utah/uic-inventory/issues/259) and in preparation for launching the [UIC Inventory Web Application](https://github.com/uic-utah/uic-inventory), deprecate the attribute rule (and associated code) for the UIC Geodatabase FacilityID field. This includes:

- [idCalculation.js](https://github.com/uic-utah/uic-attribute-rules/blob/master/src/rules/arcade/facility/idCalculation.js), and
- Associated lines of code within [facility.py](https://github.com/uic-utah/uic-attribute-rules/blob/master/src/rules/facility.py), specifically:
  - [code block 1](https://github.com/uic-utah/uic-attribute-rules/blob/5ae9f8db62ebc712b5d27b31ab5bffc0956b86d9/src/rules/facility.py#L21-L23)
  - [code block 2](https://github.com/uic-utah/uic-attribute-rules/blob/5ae9f8db62ebc712b5d27b31ab5bffc0956b86d9/src/rules/facility.py#L44)

**Comment by steveoh** [comment]
This seems like a great issue that you can perform on your own. What do you think?

**Comment by rsparker-utah** [comment]
Sure, I'd be happy to give this a try. To make sure I go about it correctly, is this the proper workflow to deprecate the existing code blocks:
- Remove the following attribute rule (and associated code) for the UIC Geodatabase FacilityID field in GitHub via a pull request:
  - [idCalculation.js](https://github.com/uic-utah/uic-attribute-rules/blob/master/src/rules/arcade/facility/idCalculation.js),
  - Associated [code block 1](https://github.com/uic-utah/uic-attribute-rules/blob/5ae9f8db62ebc712b5d27b31ab5bffc0956b86d9/src/rules/facility.py#L21-L23) and [code block 2](https://github.com/uic-utah/uic-attribute-rules/blob/5ae9f8db62ebc712b5d27b31ab5bffc0956b86d9/src/rules/facility.py#L44)
 - Delete the attribute rule in the UDEQ.UICADMIN.Facility feature class within ArcPro associated with the FacilityID field (I'll just do the dev version of the database to begin with):
![image](https://github.com/uic-utah/uic-attribute-rules/assets/49735267/94942d22-c2d8-4bad-930b-e2bc79c9a9df)

Candace and I never got around to discussing how attribute rule code is managed, so I'm a little unclear how the code in GitHub is supposed to be transferred to the UIC geodatabase i.e., is GitHub just for storing the code and then the code is manually added to the UIC geodatabase via ArcPro or is there an automated process that pulls the updated code from GitHub and implements it into the UIC geodatabase?

**Comment by steveoh** [comment]
Yep, a pull request would be great. There is no automated process to transfer this code to the uic database. You basically follow the [instructions in the read me](https://github.com/uic-utah/uic-attribute-rules#attribute-rules) and run the python. 

Since you're going to manually remove the AR rule in the database there is no reason to run the python tool. Part of me thinks this project is basically reference at this point since you are not actively creating new and updating existing AR rules. It's probably best to keep them synchronized but since no one is running the tool there's not a lot of benefit.

**Comment by rsparker-utah** [comment]
Created pull requests for:

- Removing [code block 1](https://github.com/uic-utah/uic-attribute-rules/blob/5ae9f8db62ebc712b5d27b31ab5bffc0956b86d9/src/rules/facility.py#L21-L23) and [code block 2](https://github.com/uic-utah/uic-attribute-rules/blob/5ae9f8db62ebc712b5d27b31ab5bffc0956b86d9/src/rules/facility.py#L44) of [facility.py](https://github.com/uic-utah/uic-attribute-rules/blob/master/src/rules/facility.py).
- Adding a comment to [idCalculation.js](https://github.com/uic-utah/uic-attribute-rules/blob/master/src/rules/arcade/facility/idCalculation.js) to indicate it is no longer in use.


Removed the FacilityID attribute rule from UIC Geodatabase Development instance using ArcPro.

**Comment by nathankota** [comment]
@rsparker-utah can this issue be closed now that #125 and #126 are complete?

**Comment by rsparker-utah** [comment]
@nathankota I left it open for the time being as a reminder that the FacilityID attribute rule in the production instance of the UIC Geodatabase will still need to be removed once the app is able to generate the facility ID and is in production (the FacilityID attribute rule in the production instance of the UIC geodatabase is still being used in our current workflow). That being said, I have a separate note about this, so this issue could be closed if it would be helpful.

**Comment by nathankota** [comment]
I'll reference this in an issue in the inventory project's repo
